build_library/vm_image_util.sh

```diff
@@ -431,7 +437,8 @@ _disk_ext() {
         vmdk_scsi) echo vmdk;;
         vmdk_stream) echo vmdk;;
         hdd) echo hdd;;
-        vhd*) echo vhd;;
+        vhd) echo vhd;;
```

Comment:
I think the the other cases still need to be handled, e.g., `vhd_fixed`

Reply:
added the vhd_fixed too, for better readability, similar to the vmdk cases.